packages/tailwind/docs/PseudoSelectors.stories.tsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from "react"
+import classnames from "classnames"
+import { withDesign } from "storybook-addon-designs"
+import { Card } from "@kaizen/draft-card"
+import { figmaEmbed } from "../../../storybook/helpers"
+import styles from "./styles.module.scss"
+
+export default {
+  title: "Tailwind",
+  parameters: {
+    docs: {
+      description: {
+        component:
+          'Add `presets: [require("@kaizen/design-tokens").TailwindPreset]` into your tailwind config',
+      },
+    },
+    ...figmaEmbed(
+      "https://www.figma.com/file/eZKEE5kXbEMY3lx84oz8iN/%E2%9D%A4%EF%B8%8F-UI-Kit%3A-Heart?node-id=14473%3A63845"
+    ),
+  },
+  decorators: [withDesign],
+}
+
+export const TailwindPsuedoStates = () => (
+  <div className="p-12">
+    <h1 className="flex flex-col items-center text-heading-1 font-weight-heading font-family-heading text-purple-700">
+      Tailwind Psuedo states
+    </h1>
+    <div className="flex">
+      <div className="inline-flex flex-col">
+        <h2 className="text-heading-2 font-family-heading-2 font-weight-heading-2 text-purple-800 mb-12">
+          Hover
+        </h2>
+        <button className=" rounded-default border-none p-12 bg-purple-600 text-white font-family-paragraph font-weight-paragraph-bold text-paragraph rounded border-width-default  hover:bg-purple-800">
+          Hover me
+        </button>
+      </div>
+      <div className="inline-flex flex-col ml-12">
+        <h2 className="text-heading-2 font-family-heading-2 font-weight-heading-2 text-purple-800 mb-12">
+          Focus
+        </h2>
+        <button
+          className={classnames(
+            "focus:ring-default-width ring-default-color ring-offset-default-width focus:outline-none",
+            "rounded-default border-none border-width-default",
+            " p-12 bg-purple-600 text-white font-family-paragraph font-weight-paragraph-bold text-paragraph "
+          )}
```

Comment:
Is this what the consumer needs to put in for a focus ring?! 😱 
Could we not make a special class that deals with the ring as a whole (position, direction offsets, width, colour, radius)?

Reply:
Easily.

We've been pretty against using the `component` layer so far, but this could be a good use case for it.
Even if we don't, 3 TW classes feels more straightforward than 3 imported sass variables on an `after` pseudo element.

https://cs.github.com/cultureamp/murmur/blob/2439fc43b3b2fa1c9688c86b04ede8a021beb047/app/assets/stylesheets/global/ca-button.scss?q=owner%3Acultureamp+focus-ring#L112

Reply:
Yeah, most teams have been pretty vocal in not wanting to use tailwind components too. We could release it as is and then do both could be an option if people find it too verbose.